src/main/kotlin/com/ampnet/projectservice/controller/OrganizationMemberController.kt

```diff
@@ -0,0 +1,112 @@
+package com.ampnet.projectservice.controller
+
+import com.ampnet.projectservice.controller.pojo.request.UpdateOrganizationRoleRequest
+import com.ampnet.projectservice.controller.pojo.response.OrganizationMembershipResponse
+import com.ampnet.projectservice.controller.pojo.response.OrganizationMembershipsResponse
+import com.ampnet.projectservice.grpc.userservice.UserService
+import com.ampnet.projectservice.service.OrganizationMemberService
+import com.ampnet.projectservice.service.pojo.OrganizationMemberServiceRequest
+import mu.KLogging
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RestController
+import java.util.UUID
+
+@RestController
+class OrganizationMemberController(
```

Comment:
Rename to OrganizationMembershipController. Do the same for the test. Don't forget to change in docs after renaming the test.